batan-client/store/games.js

```diff
@@ -0,0 +1,21 @@
+var game_mock = {
+  "game 1": {"id": "longUID", "board": {}},
+  "game 2": {"id": "longUID2", "board": {}}
+}
```

Comment:
Is this for the client having two games going at once? Do we have to limit to two?

Reply:
This is a mock so I could check the navbar; the idea is to have events populate the whole data structure with limits imposed explicitly, if at all